packages/forklift-console-plugin/src/modules/Providers/modals/EditProviderVDDKImage/EditProviderVDDKImage.tsx

```diff
@@ -108,9 +108,10 @@ const EditProviderVDDKImage_: React.FC<EditProviderVDDKImageProps> = (props) =>
       model={ProviderModel}
       body={
         <Trans t={t} ns="plugin__forklift-console-plugin">
-          {
-            'Virtual Disk Development Kit (VDDK) container init image path.<br><br>Note: It is strongly recommended to specify a VDDK init image to accelerate migrations.'
-          }
+          Virtual Disk Development Kit (VDDK) container init image path.
+          <br />
```

Comment:
```suggestion
          Virtual Disk Development Kit (VDDK) container init image path.<br />
```
the user will see only one EOL

Reply:
lint:fix switches back to
<br/>
<br/>

Reply:
ok